Makefile

```diff
@@ -1,5 +1,5 @@
 MOD_FLAGS := $(shell (go version | grep -q -E "1\.(11|12)") && echo -mod=vendor)
-CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | xargs -I{} basename {}))
+CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | grep -v opm/... | xargs -I{} basename {}))
```

Comment:
Do we want to continue to use `go list` like this? Maybe we should just explicitly list the things we build rather than rely on using this command and ignoring part of the result? Or at least should we have a heuristic for determining it (like only look at the packages in the root of `./cmd/`?

@dinhxuanvu Any thoughts on this? I know you most likely have a similar problem with the bundle build command you are adding.

Reply:
So the plan for me is to separate the functional code of my bundle PR to have it as a package living under `/pkg` and the cobra command code (the rest) will live under `/opm/bundle/...` and it will consume the bundle package.

Reply:
Why do we want to omit `opm` from the build?

Reply:
@dinhxuanvu I thought we wanted to expose the bundle build as a command that wasn't built as part of any of our existing utilities? and then we create another pr against operator-sdk that consumed that cobra thing? We can create our own test utility that can also use that cobra thing but we won't publish it, just use it to test. but I was expecting it to NOT be part of `opm`.

@ecordell I think he just meant that he doesn't want the subcommand packages to be part of the list of things that should be called by `go build`. Those packages don't have `main.go` files so the command that's in the makefile now will throw an error.

Reply:
I think this will do what we want:

```
CMDS  := $(addprefix bin/, $(shell ls ./cmd))
```

Reply:
@kevinrizza You are correct. The ultimate usage for bundle library would be on SDK side. The users won't use bundle commands as a part of `opm` binary. I'm planning to add bundle commands with `hidden` flag on cobra. So we know they are there and available for internal use but they won't show up in the list of available commands.